project/src/main/webapp/scripts/editQuestionOutlinesScripts.js

```diff
@@ -157,13 +151,30 @@ function crop(lx, ly, rx, ry) {
   cropCanvas.drawImage(pdfCanvas, cropX, cropY, cropWidth, cropHeight, 0, 0, cropWidth, cropHeight);
 }
 
+// onclick called when "Add Question" button is clicked
+function submitBoxButton() {
+  const assignmentKey = getUrlVars()['assignment-key'];
+  var qName = document.getElementById('question-name').value;
+  var qPoints = document.getElementById('question-points').value;
+  var qType;
+  if (document.getElementById('multiple-choice').checked){
+      qType="multiple-choice";
+  } else if (document.getElementById('short-answer').checked){
+      qType="short-answer";
+  } else {
+      qType="other";
+  }
 
-// function cropAllPDFs() {
-//     fetch('/manageBox').then(response => response.json()).then(data => {
-
-//         console.log(data);
-
-//         // crop all the pdfs based off the coordinates
-//     });
+  if (lx!=undefined && rx!=undefined && qName!=""){
+    var url = '/manageBox?'+"&qName="+qName+"&qType="+qType+"&qPoints="+qPoints+'&assignment-key='+assignmentKey+'&lx='+lx+'&ly='+ly+'&rx='+rx+'&ry='+ry; 
```

Comment:
You can directly insert variables into strings with ${variableName} and you don't have to use '+'. Just use a back tick (`) right above the tab key instead of single or double quotes.

For example:

var url = `manageBox?qName=${qName}&qType=${qType}&qPoints=${qPoints}&assignment-key=${assignmentKey}&lx=${lx}&ly=${ly}&rx=${rx}&ry=${ry}`;

Reply:
Thanks! Implemented